framework/base/Security.php

```diff
@@ -440,7 +432,7 @@ public function validateData($data, $key, $rawHash = false)
      *
      * @param integer $length the number of bytes to generate
      * @return string the generated random bytes
-     * @throws Exception on failure.
+     * @throws \Exception on failure.
```

Comment:
this is not true, it throws a `yii\base\Exception` and this class is in `yii\base` so original is fine.